### PR TITLE
workOrders.draft endpoint: MVP

### PR DESCRIPTION
**Pull request by WouterSioen**
Part of https://github.com/teamleadercrm/api/pull/290 that will be build first in isolation

### CODE REVIEW

**Review by JordyDeGroeve — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

File: src/apiary.apib [hunk 1]
```diff
@@ -77,6 +77,10 @@ HOST: https://api.teamleader.eu
 
 :[Timers](./06-time-tracking/timers.apib)
 
+# Group Work Orders
+
+ :[Work Orders](./01-general/work-orders.apib)
```
Comment (dietrichm) [inline src/apiary.apib]: B: space at the start of the line (causing weird issues in the compiled file).

File: src/01-general/work-orders.apib [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Work Orders [/workOrders]
+
+ ### workOrders.draft [POST /workOrders.draft]
```
Comment (dietrichm) [inline src/01-general/work-orders.apib]: B: spaces at the start of all lines (causing weird issues in the compiled file). The request is not recognized as such.

screen shot 2019-02-19 at 10 36 13

**Review by dietrichm — CHANGES_REQUESTED** [review]

**Review by dietrichm — DISMISSED** [review]

**Comment by dietrichm** [comment]
✅ Checked

**Review by dcsg — DISMISSED** [review]

**Review by JorenSaey — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by dietrichm — COMMENTED** [review]
Q: just now I'm seeing `starts_at` is not included yet? This is a required property in the proposal denoting the datetime the work is going to be or was carried out.

**Comment by WouterSioen** [comment]
Will add it tomorrow

**Comment by WouterSioen** [comment]
Added the missing `starts_at` field (which was not in the main api proposal at the time I made this one)

**Review by JordyDeGroeve — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

**Pull request merged**